Gemfile.lock

```diff
@@ -130,7 +130,7 @@ GEM
       rubocop-ast (>= 1.38.0, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 2.4.0, < 4.0)
-    rubocop-ast (1.38.0)
```

Comment:
Please rebase your branch, as it currently includes unrelated changes.
Please make sure your PR includes only relevant changes and only one integration at a time